### PR TITLE
Wip lesson

### LINKED ISSUE #1: new content in progress
in progress - using [accelrated sinatra views](https://github.com/learn-co-curriculum/accelerated-sinatra-views) as reference

### PR DESCRIPTION
**Pull request by ipc103**
fixes #1 -  ready for review @aviflombaum @dfenjves @SophieDeBenedetto @AnnJohn @deniznida

### CODE REVIEW

**Comment by AnnJohn** [comment]
@SophieDeBenedetto Can you review? Deadline- 9/11?

**Comment by SophieDeBenedetto** [comment]
yes

File: README.md [hunk 1]
```diff
@@ -1,8 +1,124 @@
 # Sinatra Views: Using ERB
 
-## Outline
 
-1. Explain the concept of a templating language and how we can use it in Sinatra.
-2. Explain the substitution, or `<%=` tag and why we use it.
-3. Explain the scripting tag, or `<%` tag, and what it's purpose is.
-4. Demonstrate examples of iteration and if statements in views. 
+## Overview
```
Comment (SophieDeBenedetto) [inline README.md]: add objectives

File: README.md [hunk 2]
```diff
@@ -1,8 +1,124 @@
...
+
+Most major web frameworks provide some means of view templating, i.e. allowing the view, in our case an HTML document, to be constructed using a combination of HTML and Ruby code.
+
+This allows us to greatly reduce duplication of HTML, as well as generate HTML that can change based on the data that is available. This is how Facebook can support 2 billion users - they create one template for the profile page which gets filled in with data from their servers representing each user. 
+
+For this exercise, we will be using the ERB templating engine, which comes standard with every Ruby installation.
+
+## Setup
+
+For this code-along, for and clone this repository to get started. Run `bundle install` from the command line to install any needed gems. Our starter code has a basic Sinatra application - you can preview it by running `shotgun` from the command line and opening `http://localhost:9393/` in your browser. Let's check out what we have so far.
```
Comment (SophieDeBenedetto) [inline README.md]: "for" -> "fork"

File: README.md [hunk 3]
```diff
@@ -1,8 +1,124 @@
...
+</ul>
+```
+
+Notice that we use the substitution tag to display the value of the inner `square` variable. Again, imagine that Facebook wants to print out all of your wall posts on your profile page. They could store the posts in an array called `wall_posts`. Add the following line of code to your `index.erb` file: 
+
+```ruby
+<% wall_posts = ["First post!", "Second post!", "Hello, it's your mother, why don't you ever call me?"] %>
+```
+
+Here, wer'e defining a variable called `wall_posts` and assigning it's value to an array of strings. Now, we can iterate through our wall_posts array and create a new `li` for each one.
```
Comment (dfenjves) [inline README.md]: typo "we're"

File: README.md (hunk 1 above)
Comment (ipc103) [inline README.md]: updated

**Comment by ipc103** [comment]
fixed typos / added objectives - ready for merge @SophieDeBenedetto

**Pull request merged**